sdk-extensions/incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/fileconfig/DeclarativeConfigContext.java

```diff
@@ -51,6 +59,19 @@ public void setMeterProvider(MeterProvider meterProvider) {
     this.meterProvider = meterProvider;
   }
 
+  Resource getResource() {
+    // called via reflection from io.opentelemetry.sdk.autoconfigure.IncubatingUtil
+    return resource;
```

Comment:
Maybe throw an exception if `getResource` is called before `setResource.

Reply:
good idea - added